dropwizard-metrics/3.1/src/main/java/com/wavefront/integrations/metrics/WavefrontReporter.java

```diff
@@ -337,8 +337,30 @@ public WavefrontReporter build(String proxyHostname, int proxyPort) {
           includeJvmMetrics,
           disabledMetricAttributes);
     }
-  }
+  
+
 
+  /**
+   * Builds a {@link WavefrontReporter} with the given properties, sending metrics using the given
+   * {@link WavefrontSender}.
+   *
+   * @param Wavefront a {@link WavefrontSender}.
+   * @return a {@link WavefrontReporter}
+   */
+  public WavefrontReporter build(WavefrontSender wavefrontSender) {
+    return new WavefrontReporter(registry,
+            wavefrontSender,
+            clock,
+            prefix,
+            source,
+            pointTags,
+            rateUnit,
+            durationUnit,
+            filter,
+            includeJvmMetrics,
+            disabledMetricAttributes);
+  }
+}
   private static final Logger LOGGER = LoggerFactory.getLogger(WavefrontReporter.class);
```

Comment:
Probably not your issue but can you move the logger to the top of the class? Thanks.